code/game/objects/items/reagent_containers/pill.dm

```diff
@@ -158,6 +158,11 @@
 	list_reagents = list(/datum/reagent/medicine/tramadol = 15)
 	pill_id = 7
 
+/obj/item/reagent_containers/pill/isotonic
+	pill_desc = "A pill with an isotonic solution inside. Used to stimulate blood regeneration."
+	list_reagents = list(/datum/reagent/consumable/nutriment = 10, /datum/reagent/iron = 10, /datum/reagent/consumable/sugar = 5)
```

Comment:
Pills are 15u, as a rule. I'd either go 5/5/5 or just a 7.5 split on nutri/iron. Call it fortified nutrition or something, maybe.

Reply:
didnt know about the pill cap, thanks